### PR TITLE
Improve the CreateContext documentation

### LINKED ISSUE #210: Make CreateContext summary more accessible and friendly to new users
The top-level summary and remarks of [CreateContext](https://open-ephys.github.io/onix1-bonsai-docs/api/OpenEphys.Onix1.CreateContext.html) is a bit jargony. It might require additional description or more external links.

Screenshot 2024-08-13 133458

Also, cref="Driver" anchors to the top of document for some reason, gets transformed into:

``<a class="xref" href="OpenEphys.Onix1.CreateContext.html">Drive</a>``
I think this is something that requires change in the src code repo, not the docs repo.

Relevant lines:

https://github.com/open-ephys/onix-bonsai-onix1/blob/7ac700fb8a696d4d9f8d97c9d1b85765b9951ca6/OpenEphys.Onix1/CreateContext.cs#L8-L11

### PR DESCRIPTION
**Pull request by jonnew**
- Move jargon from summary to remarks.
- Expand remarks to explain jargon a bit and provide link to ONI spec.
- Fixes #210

### CODE REVIEW

File: OpenEphys.Onix1/CreateContext.cs [hunk 1]
```diff
@@ -6,9 +6,18 @@
 namespace OpenEphys.Onix1
 {
     /// <summary>
-    /// Creates a <see cref="ContextTask"/> to orchestrate a single ONI-compliant controller
-    /// using the specified device driver and host interconnect.
+    /// Creates a <see cref="ContextTask"/> to that orchestrates data acquisition for single ONIX.
```
Comment (cjsha) [inline OpenEphys.Onix1/CreateContext.cs:9]: "to that orchestrates" => "to orchestrate" or "that orchestrates"

File: OpenEphys.Onix1/CreateContext.cs [hunk 2]
```diff
@@ -6,9 +6,18 @@
...
     /// </summary>
+    /// <remarks>
+    /// ONIX is built on top of the <see href="https://open-ephys.github.io/ONI/">ONI</see> hardware specification and API.
+    /// One of ONI's requirements is the creation of a context that holds information that used for communication between the
+    /// computer and ONI-compliant hardware. The context holds data such as the device driver that will be used to communicate
+    /// with hardware, what devices (e.g. headstages) are currently attached to the hardware, how often data should be read
+    /// from hardware, the run state of the system, etc. <see cref="ContextTask"/> creates a ONI context for a single ONIX
```
Comment (cjsha) [inline OpenEphys.Onix1/CreateContext.cs:16]: "a ONI context" => "an ONI context"

> <see cref="ContextTask"/> creates a ONI context

I think this is more accurate? 

> CreateContext creates \<see cref="ContextTask"/\> which is an ONI context.

Comment (cjsha) [reply OpenEphys.Onix1/CreateContext.cs:16]: Also I've taken for granted that I should put two periods when etc. terminates a sentence, not sure what the correct method is here

**Review by cjsha — APPROVED** [review]
This PR is approved but please go through and consider all the comments - I'm not sure it requires another review cycle after you address them (though you can re-request me if you want)

File: OpenEphys.Onix1/CreateContext.cs (hunk 2 above)
Comment (jonnew) [reply OpenEphys.Onix1/CreateContext.cs:16]: I think if a sentence ends in abbreviation, a second period is not needed (and is incorrect).

Comment (cjsha) [reply OpenEphys.Onix1/CreateContext.cs]: good to know, thanks

**Pull request merged**